api/storage/storage.py

```diff
@@ -0,0 +1,43 @@
+from boto3wrapper.wrapper import get_session
+from os import environ
+from logger import log
+from uuid import uuid4
+
+
+def get_object(record):
+
+    if environ.get("AWS_LOCALSTACK", False):
+        client = get_session().resource("s3", endpoint_url="http://localstack:4566")
+    else:
+        client = get_session().resource("s3")
+
+    obj = client.Object(record["s3"]["bucket"]["name"], record["s3"]["object"]["key"])
+    try:
+        body = obj.get()["Body"].read()
+        log.info(
+            f"Downloaded {record['s3']['object']['key']} from {record['s3']['bucket']['name']} with length {len(body)}"
+        )
+        return body
+    except Exception as err:
+        log.error(
+            f"Error downloading {record['s3']['object']['key']} from {record['s3']['bucket']['name']}"
+        )
+        log.error(err)
+        return False
+
+
+def put_file(file):
+
+    client = get_session().resource("s3")
```

Comment:
Would it make sense to have the same `if environ.get("AWS_LOCALSTACK", False):` test here that you've got for `get_object` above so that you could test `put_file` locally?